test/containers/trends/CBGSlicesAnimationContainer.test.js

```diff
@@ -81,7 +81,7 @@ describe('CBGSlicesAnimationContainer', () => {
       const instance = wrapper.instance();
       sinon.spy(instance, 'mungeData');
       expect(instance.mungeData.callCount).to.equal(0);
-      wrapper.setProps({ data: [{ id: 'a1b2c3', msPer24: 0, value: 90 }] });
+      wrapper.setProps({ data: [{ id: 'a2b3c4', msPer24: 6000, value: 180 }] });
```

Comment:
I will have a look - maybe it was a merge thing as I didn't intentionally change it ??